### DISCUSSION TITLE
How to Cutomize right side bar

### DISCUSSION THREAD

**Discussion by agusramdan** [gityopie/odoo-addons#210] [General]
Hi,

I wondering to know how to customize right sidebar google map widget for my module.

This is my module implentation https://github.com/agusramdan/odoo-addons-fleet/blob/main/fleet_vehicle_google_map/views/fleet_vehicle.xml

Please advice.

Regards

**Comment by gityopie** [comment]
Hi,

Only two type of fields you can use on the map sidebar, `Char` and `Many2one`
If you want to customise it, you will need you modify the `xml` and the `javascript` part by yourself as much as you need to
